app/vmui/packages/vmui/src/components/Chart/Line/Legend/LegendConfigs/LegendConfigs.tsx

```diff
@@ -49,7 +65,7 @@ const LegendConfigs: FC<Props> = ({ data }) => {
       <div className="vm-legend-configs-item vm-legend-configs-item_switch">
         <span className="vm-legend-configs-item__label">Common Labels</span>
         <Switch
-          label={hideDuplicates ? "Hide" : "Show"}
+          label={`${isCompact ? "Common labels" : hideDuplicates ? "Hide" : "Show"}`}
```

Comment:
Can we change the toggle to `Hide common labels` and reverse its behavior?
To me, `Hide common labels` is easier to understand than `Common labels`.

Reply:
@hagen1778 Should we also apply this to "Statistics" when implementing "Hide common labels" so that adjacent toggles follow the same logic?